docroot/sites/transportation.uiowa.edu/modules/transportation_calculator/src/Form/CostCalculatorForm.php

```diff
@@ -0,0 +1,220 @@
+    $yearly = $monthly * 12;
+    $vanpool = (10.44 + .2252 * $form_state->getValue('distance') * 21) * 12 / 6;
+    $upass = 15 * 12;
+
+    $form['results'] = [
+      '#type' => 'container',
+      '#attributes' => [
+        'class' => [
+          'results-wrapper',
+        ],
+      ],
+      'costs' => [
+        '#type' => 'container',
+        '#attributes' => [
+          'class' => [
+            'costs',
+          ],
+        ],
+        'monthly' => [
+          '#type' => 'container',
+          '#attributes' => [
+            'class' => [
+              'costs-monthly',
+            ],
+          ],
+          'item' => [
+            '#type' => 'item',
+            '#title' => $this->t('Monthly commute costs'),
+            '#markup' => $this->t('<span>@monthly</span>', [
+              '@monthly' => number_format($monthly, 2),
+            ]),
+            '#field_prefix' => '$',
+          ],
+        ],
+        'yearly' => [
+          '#type' => 'container',
+          '#attributes' => [
+            'class' => [
+              'costs-yearly',
+            ],
+          ],
+          'item' => [
+            '#type' => 'item',
+            '#title' => $this->t('Yearly commute costs'),
+            '#markup' => $this->t('<span>@yearly</span>', [
+              '@yearly' => number_format($yearly, 2),
+            ]),
+            '#field_prefix' => '$',
+          ],
+        ],
+      ],
+      'savings' => [
+        '#type' => 'container',
+        '#attributes' => [
+          'class' => [
+            'savings',
+          ],
+        ],
+        'table' => [
+          '#type' => 'table',
+          '#caption' => $this->t('Depending on your distance from campus, some modes may not apply.'),
+          '#header' => [
+            $this->t('Mode of Transportation'), $this->t('Cost Per Year'), $this->t('Yearly Savings'),
+          ],
+          '#rows' => [
+            [
+              'mode' => $this->t('CAMBUS'),
+              'cost' => '$' . number_format(0, 2),
+              'savings' => '$' . number_format($yearly, 2),
+            ],
+            [
+              'mode' => $this->t('Parking'),
+              'cost' => '$' . number_format($yearly, 2),
+              'savings' => '$' . number_format(0, 2),
+            ],
+            [
+              'mode' => $this->t('Vanpool'),
+              'cost' => '$' . number_format($vanpool, 2),
+              'savings' => '$' . number_format($yearly - $vanpool, 2),
+            ],
+            [
+              'mode' => $this->t('Carpool'),
+              'cost' => '$' . number_format($yearly / 2, 2),
+              'savings' => '$' . number_format($yearly / 2, 2),
+            ],
+            [
+              'mode' => $this->t('380 Express'),
+              'cost' => '$690.00',
+              'savings' => '$' . number_format($yearly - 690),
```

Comment:
Do we want the decimals here? `number_format($yearly - 690, 2)`

Reply:
Yes! Thanks.